spec/models/post_spec.rb

```diff
@@ -1,5 +1,29 @@
 require 'rails_helper'
 
 RSpec.describe Post, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  subject { Post.new(title: 'First Post', text: 'Content Post', user_id: 1) }
+
+  before { subject.save }
+
+  it 'Title should be present' do
+    subject.title = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'Title should have a length less than 250 char' do
+    subject.title = 'a' * 251
+    expect(subject).to_not be_valid
+  end
+
+  it 'Title should be First Post' do
+    expect(subject.title).to eq 'First Post'
+  end
+
+  it 'Text should be Content Post' do
+    expect(subject.text).to eq 'Content Post'
+  end
+
+  it 'user_id should be 1' do
+    expect(subject.user_id).to eq 1
+  end
 end
```

Comment:
- [ ] Kindly would you please solve your fail tests because some of them are not passing.

![image](https://user-images.githubusercontent.com/65068771/172974130-1e38dde6-be0a-4193-a421-462fd1547f93.png)